anya/roman_numeral.rb

```diff
@@ -0,0 +1,36 @@
+class RomanNumeral
```

Comment:
I don't think this isn't optimization, rather it's design. Seems better to init the class with the setup data (numeral constants) and have the methods represent the behavior based on the input. So for e.g. if I have 4 numbers to convert, I don't create 4 instances of a class.